### PR TITLE
rethinkdb: add test

### PR DESCRIPTION
**Pull request by xu-cheng**

### CODE REVIEW

File: Library/Formula/rethinkdb.rb [hunk 1]
```diff
@@ -75,6 +73,11 @@ def plist; <<-EOS.undent
     </plist>
     EOS
   end
+
+  test do
+    shell_output("#{bin}/rethinkdb create -d test")
```
Comment (xu-cheng) [inline Library/Formula/rethinkdb.rb]: CI test failed when I use `system`. It appears that in verbose mode, `system` call would hang out indefinitely. Log is in below:

```
info: Our machine ID: 7581c949-a486-4c78-b802-43b01cebeb08
Previous message may not have been written to the log file (cannot flush stderr: Operation not supported).
info: Created directory 'test' and a metadata file inside it.
Previous message may not have been written to the log file (cannot flush stderr: Operation not supported).
```

I think the relate code is [here](https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/formula.rb#L659-L675). The `system` function doesn't handle stderr well. @jacknagel

Comment (xu-cheng) [inline Library/Formula/rethinkdb.rb]: If the maintainers think it's a trivial problem, we can move on. I already work around it using `shell_output`.

Comment (jacknagel) [inline Library/Formula/rethinkdb.rb]: That error message must come from rethinkdb, seems more like a problem on that side than our logging code (which simply redirects stderr to a file).

Comment (xu-cheng) [inline Library/Formula/rethinkdb.rb]: Ok, [It is a rethinkdb's bug](https://github.com/rethinkdb/rethinkdb/commit/8a0d68f76c3c772b5be6970f08a8ad75a957733e). Either way, this PR is ready to be merged.

**Pull request closed without merging**